contrib/adaptive-compression/Makefile

```diff
@@ -6,6 +6,15 @@ ZSTDCOMP_FILES   := $(ZSTDDIR)/compress/*.c
 ZSTDDECOMP_FILES := $(ZSTDDIR)/decompress/*.c
 ZSTD_FILES  := $(ZSTDDECOMP_FILES) $(ZSTDCOMMON_FILES) $(ZSTDCOMP_FILES)
 
+# Define *.exe as extension for Windows systems
+ifneq (,$(filter Windows%,$(OS)))
+EXT =.exe
```

Comment:
If you want to keep `$(EXT)` you should use it for all the binaries. IMO, you can probably just drop the windows compatibility and write

```
MULTITHREAD_LDFLAGS=-pthread
```

and remove the `$(EXT)` from `datagen`.